x/staging/posts/types/params_test.go

```diff
@@ -183,3 +188,35 @@ func TestValidateMaxAdditionalAttributesFieldKeyLengthParam(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateReportTypesParam(t *testing.T) {
+	tests := []struct {
+		name        string
+		reportTypes interface{}
```

Comment:
Could it be changed to []string, instead?

Reply:
@dadamu Nope. We use interface to ensure the test fails if we have a different type then the expected one